node/node.go

```diff
@@ -1845,6 +1881,24 @@ func (app *App) startAPIServices(ctx context.Context) error {
 			})),
 		)
 	}
+
+	if len(app.Config.API.NodeServiceListener) > 0 {
+		app.log.Zap().Info("starting node service", zap.String("address", app.Config.API.NodeServiceListener))
+		lis, err := net.Listen("tcp", app.Config.API.NodeServiceListener)
+		if err != nil {
+			return err
+		}
+		golden := types.ATXID(app.Config.Genesis.GoldenATX())
+		logger := app.log.Zap().Named("atx-service")
```

Comment:
might be nice to change the name of the logger. i was also seeing this in the logs and it's not just atx related, but everything else related too. maybe `nodeService` or what not, and to add it to the list of known loggers so that we could adjust its verbosity from the config file